ui/pages/send/send-content/add-recipient/domain-input.component.js

```diff
@@ -127,18 +127,12 @@ export default class DomainInput extends Component {
                   </div>
                 )}
               </div>
-              <button
+              <ButtonIcon
+                name={ICON_NAMES.CLOSE}
+                ariaLabel={t('close')}
                 onClick={this.props.onReset}
                 className="ens-input__wrapper__action-icon-button"
-              >
-                <i
-                  className="fa fa-times"
-                  style={{
-                    color: 'var(--color-icon-default)',
-                  }}
-                  title={t('close')}
-                />
-              </button>
+              />
             </>
           ) : (
             <>
```

Comment:
Would be good to get a screenshot

Reply:
<img width="375" alt="DomainInput" src="https://user-images.githubusercontent.com/46655/220744037-c2ad82bb-ab53-4c97-a178-9e5b2a1047d9.png">